Source/VMware.vSphereDSC/Tests/Unit/TestHelpers/Mocks/MockData.ps1

```diff
@@ -492,3 +501,23 @@ $script:esxAgentHostManagerConfigWithNotNullAgentVmSettings = [VMware.Vim.HostEs
 $script:esxAgentHostManagerWithNotNullAgentVmSettings = [VMware.Vim.HostEsxAgentHostManager] @{
     ConfigInfo = $script:esxAgentHostManagerConfigWithNotNullAgentVmSettings
 }
+
+$script:vmHostPciPassthruSystem = [VMware.Vim.HostPciPassthruSystem] @{
+    PciPassthruInfo = @(
+        [VMware.Vim.HostPciPassthruInfo] @{
+            Id = $script:constants.PciDeviceId + '.0'
```

Comment:
You can have two different ocnstants for the two passthrough devices instead of modifying the value.